addon/components/nypr-accounts/membership-card/index.js

```diff
@@ -0,0 +1,46 @@
+import Ember from 'ember';
+import computed from 'ember-computed';
+import layout
+  from '../../../templates/components/nypr-accounts/membership-card';
+
+export default Ember.Component.extend({
+  layout,
+  orderDateSorting: ['orderDate:desc'],
+  sortedPledges: computed.sort('pledges', 'orderDateSorting'),
+  sortedSustainingPledges: computed.filterBy('sortedPledges', 'orderType', 'sustainer'),
+  sortedOneTimePledges: computed.filterBy('sortedPledges', 'orderType', 'onetime'),
+  mostRecentPledge: computed.reads('sortedPledges.firstObject'),
+  activePledges: computed.filterBy('sortedPledges', 'isActiveMember', true),
+  activeSustainingPledges: computed.filterBy(
+    'activePledges',
+    'orderType',
+    'sustainer'
+  ),
+  activeOneTimePledges: computed.filterBy(
+    'activePledges',
+    'orderType',
+    'onetime'
+  ),
+  isLapsedOneTimeMember: computed('pledges', 'activePledges', function() {
```

Comment:
this should depend on `'sortedPledges'` instead of `'pledges'` if that's what you're using to compute the value.